addons/website/static/src/snippets/s_image_gallery/options.js

```diff
@@ -490,6 +500,22 @@ options.registry.gallery = options.Class.extend({
     _getColumns: function () {
         return parseInt(this.$target.attr('data-columns')) || 3;
     },
+    /**
+     * Returns the images (or the images wrapped in their anchor if they were
+     * linked) sorted by index.
+     *
+     * @private
+     * @returns {DOMElement[]}
+     */
+    _getLinkedImages: function () {
```

Comment:
Nitpicking but not a fan of this method name (and the related documentation) because of two things:
- "Linked" does not make me think of `<a>` at all, it actually makes me think of images linked to something. "Link" instead of "Linked" or "Anchored" instead of "Linked" might be better.
- "_getXXXImages" makes me think this returns images, not anything else. "_getLinkImages" actually makes me think this would return the image element which are inside links. Definitely not "all images **including** the ones inside links, in which case the link is returned".

I would go with `getImgHolderEls`:
- Clear meaning that it returns elements
- Clear meaning that it may return the images themselves or what holds them, like `<a>` elements

The fact the only considered "holder" other than the image themselves are `<a>` elements can be part of the documentation only.